Translations/hr.po

```diff
@@ -1,24 +1,23 @@
-# Translation of DevilutionX to Croatian
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 # gogo <trebelnik2@gmail.com>, 2021.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: DevilutionX\n"
 "POT-Creation-Date: 2022-01-08 13:27+0100\n"
-"PO-Revision-Date: 2022-01-08 13:28+0100\n"
+"PO-Revision-Date: \n"
 "Last-Translator: gogo <trebelnik2@gmail.com>\n"
 "Language-Team: \n"
 "Language: hr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.3\n"
 "Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
 "%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
-"X-Poedit-SourceCharset: UTF-8\n"
-"X-Poedit-KeywordsList: _;N_;P_:1c,2\n"
-"X-Poedit-Basepath: ..\n"
-"X-Poedit-SearchPath-0: Source\n"
```

Comment:
Removing this will cause the update from source function to no longer work.

Reply:
I don't know why this Poedit removed.